met-web/src/services/engagementService/index.ts

```diff
@@ -22,16 +18,11 @@ export const getEngagement = async (
     errorCallback: (errorMessage: string) => void,
 ) => {
     try {
+        const url = replaceUrl(API.MET_GET_ENGAGEMENT, '<engagement_id>', String(engagementId));
```

Comment:
can https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals help here?

Reply:
Here I'm actually defining the template of the URL as Base/engagement/<engagement_id> in the endpoints file. Here I'm just replacing the <engagement_id> bit. The template literals will just insert the variable from the beginning. `engagemnet/${id}`.